src/libstd/f32.rs

```diff
@@ -198,6 +198,35 @@ impl f32 {
         }
     }
 
+    /// Returns a number composed of the magnitude of one number and the sign of
+    /// another.
+    ///
+    /// Equal to `self` if the sign of `self` and `y` are the same, otherwise
+    /// equal to `-y`. If `self` is a `NAN`, then a `NAN` with the sign of `y`
```

Comment:
"otherwise equal to `-self`", and not "`-y`", right? Or am I confused now?

Also: I'd rephrase the first sentence to "Returns a number composed of the magnitude of `self` and the sign of `y`". That's more explicit. 

And yes, I know the PR is already merged, but I guess we should fix this in a new PR, if I'm right about this typo.

Reply:
@LukasKalbertodt You're absolutely right. I'm happy to fix this in a followup PR. Good catch!